core/bourbon/helpers/_contrast.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+/// Programatically determines the contrast between two colors.
+///
+/// Note that the alpha channel is ignored.
+///
+/// @link https://www.w3.org/TR/WCAG20/#contrast-ratiodef
+///
+/// @argument {color (hex)} $color-1
+///
+/// @argument {color (hex)} $color-2
+///
+/// @return {number (1-21)}
+///
+/// @example scss
+///   contrast(black, white)
+///
+/// @require {function} _lightness
+///
+/// @access private
+
+@function _contrast($color-1, $color-2) {
```

Comment:
I think naming this function `_contrast-ratio` provides a bit more clarity as to what it outputs.